### ISSUE TITLE
OSVR Tracker Viewer fails because of open-scene-graph dependency

### ISSUE THREAD

**Issue by enilsen16** [OSVR/Homebrew-OSVR#7]
It looks like open-scene-graph doesn't support OS X 10.11. https://github.com/Homebrew/homebrew/issues/46372

Is there any other way to install the Tracker Viewer on OS X?

**Comment by rpavlik** [comment]
Well, openscenegraph is a required dependency: tracker viewer is a pretty simple application combining OSVR and OSG. The options are presumably either manually building OSG to work around the build issue you linked to, or resolve the upstream issue. Given that openscenegraph is an active project (as is homebrew) it's probably safe to assume that 10.11 fixes will arrive to those projects at some point if you can't do it. (Unfortunately, lacking a Mac myself, I can't do it.)

**Comment by enilsen16** [comment]
Homebrew has a PR in the works - https://github.com/Homebrew/homebrew/pull/46776

**Comment by gching** [comment]
@enilsen16 Building OSG manually worked for me and I got Tracker Viewer working. Try that if you can't wait :>

**Comment by d235j** [comment]
@gching @enilsen16 which source code did you use, latest from repo? If so, can you try doing `brew install open-scene-graph --HEAD`? This should install the latest version of OSG from source.

**Comment by gching** [comment]
@d235j I used specifically OSG 3.4.0 back in December, but running into other problems such as not being able to detect the HMD so can't really test it out. I'll try building it with the latest version when I can!

**Comment by enilsen16** [comment]
Yeah if `brew install open-scene-graph --HEAD` works we should update the formula. :smile:
